src/sentry/static/sentry/app/components/header/index.jsx

```diff
@@ -33,7 +70,16 @@ const Header = React.createClass({
             <a href="/" className="logo">{logo}</a>
           }
           <OrganizationSelector organization={this.getOrganization()} className="pull-right" />
+
           <StatusPage className="pull-right" />
+          { percentage < 100 ?
+            <div className="onboarding-progress-bar" onClick={this.toggleTodos}>
+              <div className="slider" style={style} ></div>
+              { this.state.showTodos ? <div className="dropdown-menu"><TodoList onClose={() => {this.setState({showTodos:false});}} /></div> : null }
```

Comment:
The `onClose` prop assignment can be rewritten using [bind](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind):

```javascript
<TodoList onClose={this.setState.bind(this, {showTodos:false})}/>
```